odoo/addons/base/models/ir_attachment.py

```diff
@@ -778,7 +777,7 @@ def _from_request_file(self, file, *, mimetype, **vals):
         return self.create({
             'name': filename,
             'type': 'binary',
-            'raw': file.read(),  # load the entire file in memory :(
+            'raw': raw,  # load the entire file in memory :(
```

Comment:
The comment was linked to the `.read()` instruction, let's move it with the code